RELEASE_NOTES.md

```diff
@@ -12,5 +12,17 @@ openssl<sup>1</sup> | >=v1.x
 <sup>1</sup> openssl is needed if libdrizzle-redux is compiled with support for
 SSL connections.
 
+Migration Notes
+===============
+
+The library is now built in a separate toplevel directory `build`.
+The modified build process is as follows:
```

Comment:
Do we need to update the build instructions on the README page as well?

Reply:
It is updated, cf. https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/143/commits/7c32ea5a3499311982b11e0be589586d0a019479#diff-88b99bb28683bd5b7e3a204826ead112R94